lib/src/model/getter_setter_combo.dart

```diff
@@ -233,4 +234,18 @@ mixin GetterSetterCombo on ModelElement {
   bool get readWrite => hasPublicGetter && hasPublicSetter;
 
   bool get writeOnly => hasPublicSetter && !hasPublicGetter;
+
+  Map<String, CommentReferable> _referenceChildren;
+  @override
+  Map<String, CommentReferable> get referenceChildren {
+    if (_referenceChildren == null) {
```

Comment:
also could be shortened:

```dart
return _referenceChildren ??= {
  for (var p in allParameters)
    p.name: p,
}
```

I think the `??=` style is also going to be easier to migrate to null safety, because of the non-promotability of fields.